recipes/mwparserfromhell/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mwparserfromhell" %}
+{% set version = "0.6.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2bad0bff614576399e4470d6400ba29c52d595682a4b8de642afbb5bebf4a346
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
```

Comment:
Since you are building a c-extension, you need a `build` section under `requirements that includes a c compiler. See:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L42-L45